parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -508,18 +495,58 @@ installCNI() {
     chown -R root:root $CNI_BIN_DIR
 }
 
+# this will never be called after 1.31 hopefully as daemonset will install windows.
```

Comment:
I think that you may need to update https://github.com/Azure/AgentBaker/blob/7342099911c73b0c8c4ed026e54e22b384f15a8a/parts/windows/kuberneteswindowssetup.ps1#L390 and https://github.com/Azure/AgentBaker/blob/dev/staging/cse/windows/azurecnifunc.ps1#L1 for Windows.
Suggest passing empty value to `$global:VNetCNIPluginsURL` and updating Install-VnetPlugins to handle the empty URL when it does not need to install Windows Azure CNI.